webapp/sources/pom.xml

```diff
@@ -349,7 +350,7 @@ limitations under the License.
     <rudder-major-version>5.1</rudder-major-version>
     <rudder-version>5.1.0~alpha1-SNAPSHOT</rudder-version>
 
-    <scala-version>2.12.7</scala-version>
+    <scala-version>2.12.8</scala-version>
```

Comment:
the parent-pom has <scala-version>2.12.6</scala-version>

we should have only one version at the same time to avoid hair pulling

Reply:
This is parent-pom file, or am I missing something ?